frontend/src/metabase/visualizations/components/settings/ColumnItem/ColumnItem.tsx

```diff
@@ -43,6 +47,7 @@ export const ColumnItem = ({
       isDraggable={draggable}
       onClick={onClick}
       data-testid={draggable ? `draggable-item-${title}` : null}
+      data-enabled={!!onRemove}
```

Comment:
One of the larger sins introduced in this PR. I know we don't typically use these, but it made targeting disabled columns via CSS so much easier. In the future, it would be good to pass elements to the sorting components, rather than just objects used to render these column items.